src/swaggerUIPlugin.ts

```diff
@@ -34,11 +34,11 @@ const swaggerUI =
                   content="SwaggerUI"
                 />
                 <title>SwaggerUI</title>
-                <link rel="stylesheet" href="https://unpkg.com/swagger-ui-dist@4.16.1/swagger-ui.css" />
+                <link href=" https://cdn.jsdelivr.net/npm/swagger-ui@5.20.0/dist/swagger-ui.min.css " rel="stylesheet">
```

Comment:
Also, I'm pretty sure that those spaces inside the double quotes aren't intentional?

Reply:
Fixed 👍🏼